src/WhoopsTracker.php

```diff
@@ -31,7 +30,7 @@ public function report(\Exception $exception)
             ->put('request', $this->requestToArray(request()))
             ->put('trace', $this->exceptionToArray($exception))
             ->put('config', config()->all())
-            ->put('queries', DB::getQueryLog())
+            ->put('environment', $_ENV)
```

Comment:
Are we no longer interested in the query log?

Reply:
We are, but we can't get to it. The main purpose of LWT is production environments in which the query log is disabled. Enabling the query log in our package adds huge memory usage which is not preferable on production environments. So since we simply can't get to it efficiently, I thought we should let it go.